install.js

```diff
@@ -0,0 +1,21 @@
+const { bin } = require('./index.js');
+
+async function install () {
```

Comment:
Could you add a comment quickly explaining this approach. It took me a second to understand what it was that this was doing.... ie: the post-install script is calling the binary wrapper so that it forces it to download it upon install.

Reply:
Lgtm otherwise.

Reply:
Comment has been added @dpgraham !